soc/xtensa/intel_adsp/common/include/intel_adsp_hda.h

```diff
@@ -458,7 +458,7 @@ static inline void intel_adsp_hda_clear_buffer_interrupt(uint32_t base, uint32_t
  * @param regblock_size Register block size
  * @param sid Stream ID
  *
- * @retval buffer segment ptr
+ * @retval interrupt status
```

Comment:
In my opinion this should have been done in the previous commit. And regarding the size parameter, I thought it would be nice to write something like `size of ...`.